httpclient-jetty/src/main/java/io/fabric8/kubernetes/client/jetty/JettyHttpClient.java

```diff
@@ -76,19 +78,37 @@ public DerivedClientBuilder newBuilder() {
 
   @Override
   public CompletableFuture<HttpResponse<AsyncBody>> consumeLines(
-      HttpRequest originalRequest, BodyConsumer<String> consumer) {
-    throw new UnsupportedOperationException("Not supported by the Jetty client");
+      HttpRequest originalRequest, AsyncBody.Consumer<String> consumer) {
+    final var request = toStandardHttpRequest(originalRequest);
+    final var future = new JettyAsyncResponseListener(request) {
+
+      final StringBuilder builder = new StringBuilder();
```

Comment:
I'm not sure about the threading model of jetty - will it always be the same websocket thread making the onContent calls?

Reply:
It wouldn't need to be volatile, it would need to be a StringBuffer or synchronized to account for the possibility of reordered writes.  As you say that's probably highly unlikely given the higher level serialization control.

Also JettyHttpClientTest can remove the testConsumeLines override.